apps/web/src/state/farmsV4/state/accountPositions/fetcher.ts

```diff
@@ -203,16 +199,26 @@ export const getAccountV2LpDetails = async (
   ])
 
   const farming = bCakeWrapperAddresses.reduce((acc, address) => {
-    if (!address || address === '0x') return [...acc, undefined]
-    const { result } = _farming.shift() ?? { result: undefined }
-    return [...acc, result]
+    if (!address || address === '0x') {
+      acc.push(undefined)
+    } else {
+      const { result } = _farming.shift() ?? { result: undefined }
+      acc.push(result)
+    }
+    return acc
   }, [] as Array<readonly [bigint, bigint, bigint, bigint, bigint] | undefined>)
-
   return balances
     .map((result, index) => {
       const { result: _balance = 0n, status } = result
       // LP not exist
-      if (status === 'failure') return undefined
+      if (status === 'failure') {
+        if (reserves[index].status === 'failure' && totalSupplies[index].status === 'failure') {
+          return undefined
+        }
+        throw new Error(
```

Comment:
why throw Error, they are difference request, it would happen when some of them fetch failed but some of them succ